README.md

```diff
@@ -8,6 +8,7 @@ It creates the following resources:
 * An optional internet gateway
 * An optional NAT gateway
 * An optional service gateway
+* An optional Dynamic Routing Gateway
```

Comment:
Same as above: dynamic routing gateway should be lowercase.

Reply:
changed in both adoc and md version of the file